### PR TITLE
Set first breadcrumb in config

### PR DESCRIPTION
**Pull request by crumb1e**
Allow the default first breadcrumb to be set in the config.

This will allow for an easy way to change this on any site we use this package on, and going forward 🚀

### CODE REVIEW

**Comment by crumb1e** [comment]
@m2de dw I will add documentation before merging if you approve of this change 🐸

**Review by tristanward — APPROVED** [review]
Lovely addition to the package @crumb1e

File: src/Config/maxfactor-support.php [hunk 1]
```diff
@@ -8,4 +8,5 @@
         \Symfony\Component\HttpFoundation\Response::HTTP_TEMPORARY_REDIRECT,
         \Symfony\Component\HttpFoundation\Response::HTTP_PERMANENTLY_REDIRECT,
     ],
+    'defaultBreadcrumb' => 'Home',
```
Comment (m2de) [inline src/Config/maxfactor-support.php]: Could we please call this `homeBreadcrumb` just to make it really self explanatory?

**Review by m2de — CHANGES_REQUESTED** [review]
One super minor change. Otherwise seems like a good solution. Thanks @crumb1e

File: src/Config/maxfactor-support.php (hunk 1 above)
Comment (crumb1e) [reply src/Config/maxfactor-support.php]: Yeah that makes sense, I'll get that changed.

**Review by m2de — APPROVED** [review]
Nice one. Please add to the documentation, tag a minor and add details to the release notes on github to explain what someone can/needs to do if they are upgrading from a previous version.

**Pull request merged**